lms/templates/sudo/sudo.html

```diff
@@ -0,0 +1,21 @@
+{% extends "main_django.html" %}
+{% load i18n %}
+
+{% block body %}
+<section style="margin: 0 auto; width: 480px; padding: 50px;">
+    <div class="inner-wrapper">
+        <header>
+          <h2 style="text-align: center;">{% trans "Confirm Your Password to Access the Instructor Dashboard" %}</h2>
```

Comment:
Good call. This is what I'm seeing

![screen shot 2015-07-07 at 2 28 56 pm](https://cloud.githubusercontent.com/assets/2146312/8553928/bd740306-24b4-11e5-8bc8-9a234e6f8c60.png)


Does @waheedahmed need to manually run `paver i18n_dummy` on the sandbox, or is that done already during provisioning?